ppx/ppx_mysql.ml

```diff
@@ -1,151 +1,16 @@
 open Ppxlib
+module Query = Query
+(* So the unit tests have access to the Query module *)
```

Comment:
This comment should most likely be above the line, not below.

Reply:
It was. Ocamlformat must have moved it...